src/storage.rs

```diff
@@ -187,7 +187,7 @@ pub fn write_to_datastore(
     cfg: Arc<RwLock<Config>>,
     log_name: &str,
     payload: &String,
-) -> Result<bool, StorageError<PutObjectError>> {
```

Comment:
Looks like the bool part is not returning any extra info - it always seems to return true. We can just use `()` as the Item type here.